src/glmfit.jl

```diff
@@ -115,7 +115,23 @@ function confint(obj::AbstractGLM, level::Real)
 end
 confint(obj::AbstractGLM) = confint(obj, 0.95)
 
-deviance(m::AbstractGLM)  = deviance(m.rr)
+deviance(m::AbstractGLM) = deviance(m.rr)
+
+function loglikelihood(m::AbstractGLM)
```

Comment:
@dmbates That's kind of annoying that nobody knows how to compute that stuff. I'm not happy with the fact that in 
https://github.com/JuliaStats/StatsBase.jl/pull/146 I documented `deviance` as being equal up to a constant to `-2 log L` (like other programs do), and `AIC` as being `-2 log L + 2k`, and yet ``-2 log L` isn't the same value in those two docs: in the latter, it's really `-2loglikelihood(m)`, but in the former it's some unknown quantity to me. I must be missing something obvious.